app/controllers/community/meetups/index.js

```diff
@@ -26,6 +30,6 @@ export default Controller.extend({
   lng: 0,
   zoom: 2,
   meetupsByArea: groupBy('model', 'area'),
-  sortingKeys: Object.freeze(['items.length:desc']),
+  sortingKeys: Object.freeze(['items.length:desc', 'value:asc']),
```

Comment:
I added a tiebreaker (sort in ascending order by the area name, which is called `value`).